template.yml

```diff
@@ -1426,6 +1428,82 @@ Resources:
       Dimensions:
         - Name: LambdaFunctionArn
           Value: !GetAtt ImageTransformLambdaFunction.Arn
+  WavWrapLambdaIamRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      ManagedPolicyArns:
+        - !Ref ArtifactBucketReadOnlyAccessPolicy
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+        - arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess
+      Tags:
+        - Key: Project
+          Value: Porter
+        - Key: prx:cloudformation:stack-name
+          Value: !Ref AWS::StackName
+        - Key: prx:cloudformation:stack-id
+          Value: !Ref AWS::StackId
+  WavWrapLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: lambdas/WavWrapLambdaFunction/
+      Description: >-
+        WAV wrap a source audio file
+      Environment:
+        Variables:
+          S3_DESTINATION_WRITER_ROLE: !GetAtt S3DestinationWriterRole.Arn
+      Handler: index.handler
+      Layers:
+        - !Ref AwsXraySdkLambdaLayerVersionArn
+        - !Ref NpmSharpLambdaLayerVersionArn
+        - !Ref WavefileLambdaLayerVersionArn
```

Comment:
Now that this is coming from `npm` as `prx-wavefile`, should this layer parameter name change?
Also, looks like it isn't creaed/defined yet.

Reply:
I'm going to prototype how I want to do the layer management in #80 with wavefile in this PR. I'll get that set up this morning.